### ISSUE TITLE
Derived FIelds Loader issue: `loadRelated` missing from type `store`.

### ISSUE THREAD

**Issue by Data-Nexus** [graphprotocol/graph-tooling#1463]
### Which packages are impacted by your issue?

@graphprotocol/graph-cli

### Describe the issue

Utilizing version `0.58.0 win32-x64 node-v16.20.0` of the CLI, when attempting to utilize the derived fields loader I get the following error when attempting to build:

```
Property 'loadRelated' does not exist on type '~lib/@graphprotocol/graph-ts/index/store'.  
in generated/schema.ts(7346,23)
```

In my mapping I am attempting to load the child entities and loop through:

```
  let silos = _lendingProtocol.silos.load();

  for (let i = 0; i < silos.length; i++) {
    log.info("loop # {}", [i.toString()]);
  }
```

### Reproduction

local environment

### Steps to Reproduce the Bug or Issue

1. Utilize the `.load()` function on an array for a derived field. 
2. Run `graph build`

### Expected behavior

I expected to be able loop through the array of entities after loading them to a variable.

### Platform

- OS: Windows
- NodeJS: node-v16.20.0
- `@graphprotocol/*` version(s): 0.58.0

### Subgraph GraphQL Schema

```
type LendingProtocol implements Protocol @entity {

  " All markets that belong to this protocol "
  silos: [Silo!]! @derivedFrom(field: "protocol")
  
...
} 

type Silo @entity {
  " {Smart contract address of the market} "
  id: ID!

  " Protocol that the silo belongs to "
  protocol: LendingProtocol!
...
}
```

**Comment by azf20** [comment]
thanks - @saihaj @incrypto32 is this similar to / a duplicate of https://github.com/graphprotocol/graph-tooling/issues/1449 (I think this example from @Data-Nexus also involves interfaces?)

**Comment by saihaj** [comment]
What version of `graph-ts` are you using? For this feature to work you need v0.31.0+ https://github.com/graphprotocol/graph-tooling/releases/tag/%40graphprotocol%2Fgraph-ts%400.31.0

**Comment by Data-Nexus** [comment]
I had `^0.31.0` listed in my `package.json` and after running `npm install` the issue no longer appears when I attempt to build. 

I also assisted a hacker at ETHGlobal NY ran into the same issue and he had newly installed the CLI. 

My schema does utilize an interface, although that was not the portion that was broken. The hacker at ETHGlobal only had 2 entities and we moved away from using the derived field and storing a manual array so we could loop through them.
